release.sh

```diff
@@ -11,7 +11,7 @@ stargate_version=$1
 
 for script in $(find . -type f -name 'build.sh'); do
     echo "Executing $script"
-    bash "$script" $stargate_version || break
+    bash "$script" $stargate_version
```

Comment:
Does it actually cause the top-level script to have a non-zero exit code? Should it perhaps be `|| exit 1`

Reply:
It does, I tested locally; the issue before was that the `|| break` was swallowing any non-zero exit code and then ending the script normally, so leaving it out just causes the script to fail at line 14 (what we want)

`set -e` at the top is actually what causes a non-zero exit code in any command to end execution